index.html

```diff
@@ -42,7 +42,7 @@ <h2>Netdata Agent Helm Chart</h2>
             <footer>Install and manage Netdata in your Kubernetes (k8s) Cluster!</footer>
           </hgroup>
 	  <pre>
-	  $ helm repo add netdata https://helm.my-netdata.io/
+	  $ helm repo add netdata https://netdata.github.io/helmchart/
 	  $ helm install netdata
```

Comment:
I guess the command should be **helm install reponame/helm-chart-name**
```
helm install netdata/netdata
```

Reply:
Good catch!

Fixed.